rs/pocket_ic_server/BUILD.bazel

```diff
@@ -119,6 +117,8 @@ rust_binary(
     testonly = True,
     srcs = ["src/main.rs"],
     proc_macro_deps = MACRO_DEPENDENCIES,
+    # TODO: restrict the visibility
```

Comment:
leftover TODO

Reply:
This I will leave as a reminder for me for now.

Reply:
@rumenov can you clarify the comment then? it's not clear to me why the visibility should/could be restricted, and why it's not done now

Reply:
I recommend making a ticket and mention the ticket number.